bbb-install-2.6.sh

```diff
@@ -293,7 +293,7 @@ main() {
 
     if [ -f /etc/apt/sources.list.d/nodesource.list ] &&  grep -q 16 /etc/apt/sources.list.d/nodesource.list; then
       # Node 16 might be installed, previously used in BigBlueButton
-      sudo apt-get purge nodejs
+      # We will upgrade it rather than purge plus fresh install of a newer version.
```

Comment:
This comment is a bit difficult to understand; it doesn't really make sense to say "rather than purge" here. Instead, it should explain why the configuration file is being removed in the if branch but no other changes are being made.

Perhaps something like this:
```suggestion
      # Remove the repository config. This will cause the repository to get
      # re-added using the current nodejs version, and nodejs will be upgraded.
```